cpp/mg_utility/mg_utils.hpp

```diff
@@ -143,75 +168,55 @@ std::unique_ptr<mg_graph::Graph<TSize>> GetGraphView(mgp_graph *memgraph_graph,
 
   for (auto *vertex_from = mgp::vertices_iterator_get(vertices_it); vertex_from;
        vertex_from = mgp::vertices_iterator_next(vertices_it)) {
+    if (subgraph && subgraph_node_ids.find(mgp::vertex_get_id(vertex_from).as_int) == subgraph_node_ids.end()) {
+      continue;
+    }
```

Comment:
Again, traverse only subgraph nodes, or if there is no subgraph traverse all nodes, create a lambda iterator.

Reply:
Fixed.